.circleci/config.yml

```
@@ -21,7 +21,7 @@ workflows:
           matrix:
             parameters:
               # https://hub.docker.com/r/kindest/node/tags
-              kube_version: ["1.19.11", "1.20.7", "1.21.2"]
+              kube_version: ["1.19.11", "1.20.7", "1.21.2", "1.22.5"]
```

We should add 1.23 here too since it's already out and is api compatible with 1.22.
```suggestion
              kube_version: ["1.19.11", "1.20.7", "1.21.2", "1.22.7", "1.23.4"]
```